Restrict call seek() for smooth slide

banggaoo opened the pull request:
Change video url to youtube

#### Ticket

[TICKET_NUMBER](https://vimean.atlassian.net/browse/TICKET_NUMBER)

- *Note: this is required for Vimeo staff only. If not applicable to your PR, use "N/A".*

N/A

#### Pull Request Checklist

- [ ] Resolved any merge conflicts
- [ ] No build errors or warnings are introduced
- [ ] New files are written in Swift
- [ ] New classes contain license headers
- [ ] New classes have Documentation
- [ ] New public methods have Documentation

#### Issue Summary

- *Outline the goals of the PR in technical terms. If this branch fixes a bug, state what caused the bug.*

#### Implementation Summary

- *Identify the core components that were added or changed, and briefly state the reasoning behind these changes.*

- *List any key assumptions made.*

#### Reviewer Tips

- *Provide the order in which files or components should be reviewed.* 

- *List the general components that this PR will affect.*

- *Identify any copy and pasted code with a large diff.*

#### How to Test

- *Identify necessary conditions for testing (e.g. logged out, bad connectivity).*

- *If a special type of account is needed for log in, contact a developer privately to provide credentials.*

Commit:
Restrict call seek() for smooth slide
Change video url to youtube

Jumhyn commented:
Hi @banggaoo, could you please update the PR description with a little more information about what problems this PR solves and why you made the changes you did?

banggaoo commented:
Hi this is James Lee.

I tested PlayerKit on iPhone 6S with youtube video and i feel slider is not smoothly moved. So i changed the method of call seek() operation for improve performance. The slider moving is much smoother than previously version.

Regards.


> On 31 Jul 2018, at 1:17 AM, Frederick Kellison-Linn <notifications@github.com> wrote:
> 
> Hi @banggaoo <https://github.com/banggaoo>, could you please update the PR description with a little more information about what problems this PR solves and why you made the changes you did?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/vimeo/PlayerKit/pull/40#issuecomment-408921716>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AARYXb_8DaKjBG4v1ScQnLpfARkbR55rks5uLzGegaJpZM4VmIhF>.
>

Jumhyn commented:
Thanks for the info! Would you be able to provide the code/environment info you were using to test this? Also, your use of `automaticallyWaitsToMinimizeStalling` is causing the build to fail since it is only available in iOS 10.0. PlayerKit supports back to iOS 9.0, so you’ll need to either remove that line, or wrap it in a `#available` check.

Commit:
Change video source to Apple provided
Add TouchUpIn/Outside action for getting last drag position
Improved seek operation - divide seek operation for apply last drag position
